apps/api/src/persistence/repositories/arango-command-repository-for-aggregate-root.ts

```diff
@@ -117,6 +117,7 @@ export class ArangoCommandRepositoryForAggregateRoot<TAggregate extends Aggregat
         // TODO Have these been sorted yet?
         await this.eventRepository.appendEvents(eventHistory);
 
+        // TODO this should be done atomically with the above
         await this.snapshotRepositoryForAggregate.create(entity);
```

Comment:
I think we can probably update the implementation of `create` to also append the event. Let's have a quick look now.

Reply:
I've linked to [this story](https://www.pivotaltracker.com/story/show/188918550).